ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Solaris.yml

```
@@ -0,0 +1,161 @@
+---
+###########
+# Solaris #
+###########
+
+- name: Set sudoers file
+  set_fact:
+    Sudoers_File: "/etc/opt/csw/sudoers"
+  tags: [jenkins_user, adoptopenjdk]
+
+- name: Check if pkgutil is installed
+  shell: pkgutil --version
+  ignore_errors: true
+  register: pkgutil_installed
+  tags:
+    - build_tools
+
+- name: Install pkgutil
+  shell: |
+    set -o pipefail
+    echo action=nocheck > responses.txt
+    yes all | pkgadd -d http://get.opencsw.org/now -a responses.txt
```

`http` on here makes me very nervous although I appreciate that it's due to the same reason get_uri doesn't work.
What is `action=nocheck` required for as it seems to install ok without it?
I'd change it to this to explicitly install `CSWpkgutil` from the bundle
```suggestion
    yes | pkgadd -d http://get.opencsw.org/now CSWpkgutil
```

---

Thinking some more we should possible store our own copy of this in Vendor_Files (or other comparable location) if we can't update the CA certs that pkgadd is using

---

Hmmm in fact given that the CSWpkgutil package is only about 220K gzipped I could just about accept having it stored in this repository and then copied acrosss to the machine to be installed (something I would normally NEVER advocate, but it's very small)

---

updated PTAL